src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -57,18 +57,18 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                 else if (logRecord.StateValues != null)
                 {
                     this.WriteLine("LogRecord.StateValues (Key:Value):");
-                    for (int i = 0; i < logRecord.StateValues.Count; i++)
+                    foreach (var value in logRecord.StateValues)
                     {
                         // Special casing {OriginalFormat}
                         // See https://github.com/open-telemetry/opentelemetry-dotnet/pull/3182
                         // for explanation.
-                        if (logRecord.StateValues[i].Key.Equals("{OriginalFormat}"))
-                        {
-                            this.WriteLine($"{string.Empty,-4}{"OriginalFormat (a.k.a. Body)",-RightPaddingLength}{logRecord.StateValues[i].Value}");
-                        }
-                        else
+                        var valueToTransform = value.Key.Equals("{OriginalValue}")
+                            ? new KeyValuePair<string, object>("OriginalFormat (a.k.a Body)", value.Value)
+                            : value;
+
+                        if (ConsoleTagTransformer.Instance.TryTransformTag(valueToTransform, out var result))
                         {
-                            this.WriteLine($"{string.Empty,-4}{logRecord.StateValues[i].Key,-RightPaddingLength}{logRecord.StateValues[i].Value}");
+                            this.WriteLine($"{string.Empty,-4}{result}");
```

Comment:
Lost some formatting of alignment in a number of spots. If folks feel strongly about preserving this formatting, I had it on my mind to make an override of TryTransformTag that took a format string or the likes.

Reply:
I actually prefer that we remove all alignments.
For folks who want better alignments, we might consider an option to output markdown tables (like how Benchmark.NET is doing).

Reply:
I think the current output format is very similar to YAML, maybe we can move towards that direction? 😆